index.js

```diff
@@ -51,6 +52,9 @@ var Omegle = function () {
 		}
 		request(options, function (error, response, body) {
 			if (!error && response.statusCode == 200) {
+				if(!working_servers.indexOf(url)>0){
```

Comment:
In case the user don't know anything about Omegle servers, the option to manually set server will not work. So, `omegle_node` should be smart enough to not use that broken server again and use one which worked previously. 
In the above, I am storing working servers for later use.